app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Authenticate your Kong Gateway Amazon RDS database with AWS IAM
+badge: enterprise
+content_type: how-to
+---
+
+Starting in {{site.base_gateway}} 3.3.x, you can use AWS Identity and Access Management(IAM) authentication to connect to the AWS RDS database that you use for {{site.base_gateway}}. This page describes how to use this feature to secure your database configurations and database connections.
+
+When you enable this feature, you don't need to use a password when you connect to a database instance. Instead, you use a temporary authentication token. Because AWS IAM manages the authentication externally, the database doesn't store user credentials. If you use AWS RDS for {{site.base_gateway}}'s database, you can enable this feature to your running cluster. This ensures that you don't have to store database user credentials on both the {{site.base_gateway}} (`pg_password`) and RDS database side. 
+
+## AWS IAM authentication limitations
+
+AWS IAM authentication also has some limitations. Go through each one before you use this feature in your production environment:
+
+* The number of IAM database authentication connections must be less than the maximum of 200 in a traditional {{site.base_gateway}} cluster. Establishing more connections can result in throttling. For more information, see [Recommendations for IAM database authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html#UsingWithRDS.IAMDBAuth.ConnectionsPerSecond) in the Amazon RDS user guide. 
```

Comment:
@windmgc Revised this sentence a bit to call out the "per second" part. Does this work?

Reply:
Could you please help modify the new sentence a bit on the following points? Just want to make sure that users can understand this limitation correctly:
- This does not only happens on the traditional cluster but also on single traditional nodes. Traditional cluster which is made up of multiple traditional nodes will have more possibility to hit this limitation because every node needs to establish connections to the database. If possible users should check the TPS of the connection establishment happens on their database to make sure they're not hitting this limitation.
- `only use IAM database authentication if Kong Gateway requires less than 200 new IAM database authentications per second` - this is correct, but users may not know that when will authentication happen. The authentication only happens on each connection's initialization part, after the connection is established successfully, the following queries and communication does not need authenticate.


Thanks in advance!

Reply:
@windmgc Okay, added that info you suggested :)